django/core/management/commands/makemigrations.py

```diff
@@ -50,6 +51,11 @@ def add_arguments(self, parser):
             '--check', action='store_true', dest='check_changes',
             help='Exit with a non-zero status if model changes are missing migrations.',
         )
+        parser.add_argument(
```

Comment:
I don't think adding this makes sense. Perhaps similar logic to the migration consistency check could be used.